### PR TITLE
feat: Update known chromedriver versions automatically

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The script will be automatically executed by cron service in Travis on daily basis and will create a PR as soon as there is a new version in the chromedriver xml feed

### CODE REVIEW

File: scripts/handle_chromedriver_feed.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    for version in versions:
+        result.insert(insert_pos, f"  '{version}': '{version}',")
+    with open(js_path, 'w') as f:
+        f.write('\n'.join(result))
+
+
+if __name__ == '__main__':
+    avail_versions = extract_available_versions(sys.argv[1])
+    known_versions = extract_known_versions(sys.argv[2])
+    for known_version in known_versions:
```
Comment (KazuCocoa) [inline scripts/handle_chromedriver_feed.py:69]: Can we add versions only lower than a stable release?
https://chromedriver.storage.googleapis.com/LATEST_RELEASE

For now, our map has released versions. It does not include beta ones.

Comment (mykola-mokhnach) [reply scripts/handle_chromedriver_feed.py:69]: how do I distinguish these from the xml feed?

**Comment by mykola-mokhnach** [comment]
> concept looks good, but why write a script in python?

because it's fast to write and has "batteries included"

**Comment by mykola-mokhnach** [comment]
@jlipps I was wondering whether it would be possible to move CD mapping to a separate json file, so we don't have to do with javascript

**Comment by jlipps** [comment]
> I was wondering whether it would be possible to move CD mapping to a separate json file, so we don't have to do with javascript parsing

seems reasonable

File: scripts/handle_chromedriver_feed.py (hunk 1 above)
Comment (KazuCocoa) [reply scripts/handle_chromedriver_feed.py:69]: The key name is `LATEST_RELEASE`. It seems the etag indicates the lates version's etag.
https://chromedriver.storage.googleapis.com/
```
<Contents>
<Key>LATEST_RELEASE</Key>
<Generation>1586361174590155</Generation>
<MetaGeneration>1</MetaGeneration>
<LastModified>2020-04-08T15:52:54.589Z</LastModified>
<ETag>"5b2d74232aaf7e3e284ba7f9d0ecf862"</ETag>
<Size>12</Size>
</Contents>
```

Comment (mykola-mokhnach) [reply scripts/handle_chromedriver_feed.py:69]: ok, I can change that. Although, this means that if we skip one or more releases then these could only be added manually

Comment (mykola-mokhnach) [reply scripts/handle_chromedriver_feed.py:50]: > The key name is LATEST_RELEASE. It seems the etag indicates the lates version's etag.

I've double checked that and it looks like the `LATEST_RELEASE` does not necessarily point to the most recent version. The current recent one is 83, although it is pointing to 81.

Comment (mykola-mokhnach) [reply scripts/handle_chromedriver_feed.py]: Changed to the proposed solution. Now the implementation is much simpler

File: .travis.yml [hunk 2]
```diff
@@ -1,46 +1,79 @@
...
+    language: python
+    python: '3.8'
+    env:
+      STORAGE_URL: https://chromedriver.storage.googleapis.com
+      MAPPING_SOURCE: config/mapping.json
+      PROJECT_ROOT: github.com/appium/appium-chromedriver.git
+      API_ROOT: https://api.github.com/repos/appium/appium-chromedriver
+    script:
+    - |-
+      export LATEST_RELEASE=$(curl -sf $STORAGE_URL/LATEST_RELEASE)
```
Comment (KazuCocoa) [inline .travis.yml:18]: 👍

**Pull request merged**